uav_control/LDRInfo[] ldr_infos